packages/core/src/styles/global/index.scss

```
@@ -4,6 +4,9 @@
 // FastStore UI's base styles
 @import "@faststore/ui/src/styles/global.scss";
 
+// Import components styles (includes QuickOrderDrawer)
+@import "@faststore/ui/src/styles/components.scss";
```

Stylelint (17.3.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)